src/components/Match/MatchStory.jsx

```diff
@@ -576,7 +576,7 @@ const generateStory = (match) => {
     events.push(new FirstbloodEvent(match,
       match.objectives[fbIndex].time,
       match.objectives[fbIndex].player_slot,
-      killerLog ? killerLog[0].key : null));
+      (killerLog instanceof Array && killerLog[0] ? killerLog[0].key : null)));
```

Comment:
According to this:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray

Arrays.isArray() is preferred.

Alternatively just `killerLog && killerLog[0]` is probably sufficient to handle this case